clients/src/main/java/org/apache/kafka/common/utils/SystemTime.java

```diff
@@ -21,10 +21,17 @@
 import java.util.function.Supplier;
 
 /**
- * A time implementation that uses the system clock and sleep call. Use `Time.SYSTEM` instead of creating an instance
- * of this class.
+ * A time implementation that uses the system clock and sleep call.
+ * Every kafka process has a single instance of class <code>SystemTime</code> that allows the process to interface
```

Comment:
nit: I think the mention of "process" here is more confusing than helpful. If you want to mention this is a singleton, say "Use the singleton `Time.SYSTEM`" you can do that, or just leave the javadoc as-is.

Reply:
Thank you
 I have restored it